testsuite/kubernetes/monitoring/pod_monitor.py

```diff
@@ -0,0 +1,37 @@
+"""Module implements Pod Monitor CR"""
+
+from testsuite.kubernetes import KubernetesObject
+from testsuite.kubernetes.client import KubernetesClient
+from testsuite.kubernetes.monitoring import MetricsEndpoint
+from testsuite.utils import asdict
+
+
+class PodMonitor(KubernetesObject):
```

Comment:
Why do we need to use PodMonitor insteaed of ServiceMonitor? 
I would like to see at lůeast some explanation in the PR description

Reply:
I don't have an answer, I followed documentation: https://github.com/Kuadrant/limitador/tree/main/limitador-server/kubernetes#monitoring

Reply:
I would prefer using either servicemonitor or podmonitor in all of testsuite